static/js/settings_org.js

```diff
@@ -81,6 +81,26 @@ var org_permissions = {
     },
 };
 
+exports.maybe_disable_widgets = function () {
+    if (page_params.is_admin) {
+        return;
+    }
+
+    $(".organization-box [data-name='organization-profile']")
+        .find("input, textarea, button, select").attr("disabled", true);
+
+    $(".organization-box [data-name='organization-settings']")
+        .find("input, textarea, button, select").attr("disabled", true)
+        .find(".control-label-disabled").addClass('enabled');
+
+    $(".organization-box [data-name='organization-permissions']")
+        .find("input, textarea, button, select").attr("disabled", true)
+        .find(".control-label-disabled").addClass('enabled');
```

Comment:
Does that chained `.find()` do the right thing?  I'd worry that it ends up having an ordering issue.

Reply:
(The `.control-label-disabled` things are outside the text reas).

Reply:
oops, yeah, that's wrong, I need to un-chain those